cmd/oceanbench/api.go

```diff
@@ -91,6 +91,40 @@ func apiHandler(w http.ResponseWriter, r *http.Request) {
 			fmt.Fprint(w, string(enc))
 			return
 
+		case "devices":
+			switch val {
```

Comment:
Same feedback as other PR

Reply:
Not quite sure why they appear in this PR as well. The base branch of this PR should be that other PR and these changes shouldn't show here. I will make sure to fix before merging.